pkg/ocmagenthandler/ocmagenthandler_configmap.go

```diff
@@ -0,0 +1,95 @@
+package ocmagenthandler
+
+import (
+	"context"
+	"reflect"
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+
+	ocmagentv1alpha1 "github.com/openshift/ocm-agent-operator/pkg/apis/ocmagent/v1alpha1"
+	oahconst "github.com/openshift/ocm-agent-operator/pkg/consts/ocmagenthandler"
```

Comment:
oah?

Reply:
Changed all uses of `oahconst` to `oah` in latest commit.